packages/liferay-js-publish/src/index.js

```diff
@@ -175,11 +171,53 @@ function getRemote() {
 	throw new Error('Unable to determine remote repository URL');
 }
 
+function isPrereleaseVersion(pkg) {
+	const {version} = pkg;
+
+	const parts = version.split('.');
+
+	// De facto standard for prerelease versions is; 10.0.0-alpha.1
```

Comment:
Not "de facto": it is well defined [here](https://semver.org/).

Anything with a hyphen is a prelease, and that's it. This entire function could be reduced to:

```js
const {version} = pkg;

return version.includes('-');
```

Reply:
True, because you cannot have hyphens inside major and minor version numbers.